test/extended/operators/certs.go

```diff
@@ -111,6 +142,10 @@ var _ = g.Describe(fmt.Sprintf("[sig-arch][Late][Jira:%q]", "kube-apiserver"), g
 			masters = append(masters, &nodeList.Items[i])
 		}
 
+		// etcd operator doesn't immediately remove the certificate for boostrap
+		err = cleanupEtcdCertificates(ctx, kubeClient, masters)
```

Comment:
openshift-tests cannot make lasting change to a cluster. Why is this necessary?

Reply:
Etcd-operator now keeps previous certkeys in preparation to signer/ca/key rotation, so this code will remove bootstrap certkey. It has a "garbage collector", but there is no way to trigger it immediately.

Perhaps etcd-operator should have a way to trigger it immediately - and eventually do this automatically after bootstrap?

Reply:
TODO: RewriteNodeIPs should include bootstrap IP (fetch it from secrets in openshift-etcd namespace)